docs/source/tutorials/faq.ipynb

```diff
@@ -376,6 +376,67 @@
     "While this open-source library **finds** data issues, an interface is needed to efficiently **fix** these issues in your dataset. [Cleanlab Studio](https://cleanlab.ai/studio/?utm_source=github&utm_medium=docs&utm_campaign=clostostudio) is a no-code platform to find and fix problems in real-world ML datasets. Studio automatically runs optimized versions of the algorithms from this open-source library on top of AutoML models fit to your data, and presents detected issues in a smart data editing interface. Think of it like a data cleaning assistant that helps you quickly improve the quality of your data (via AI/automation + streamlined UX)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "21f42f24",
+   "metadata": {},
+   "source": [
+    "### How can I apply filter.find_label_issues() to big datasets with limited memory? "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "089f505e",
+   "metadata": {},
+   "source": [
+    "A few tips you can try:\n",
+    "\n",
+    "- Set the argument `n_jobs = 1` to disable multiprocessing which may consume extra memory.\n",
+    "- Set the argument `multi_label = True` which runs `find_label_issues()` in a one-vs-rest fashion that uses less memory.\n",
+    "- Instead try the `LabelInspector` class in `cleanlab.experimental`. This class reads data in mini-batches and does not use much memory (you can reduce the `batch_size` to control how much). Below is an example of how to use it. This example forms mini-batches out of `labels`, `pred_probs` that are already loaded in memory but you could load mini-batches from file instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41714b51",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.experimental.label_issues_batched import LabelInspector\n",
```

Comment:
Why is the suggested response to use the `LabelInspector` object instead of `cleanlab.experimental.label_issues_batched import find_label_issues_batched`

Reply:
because people with really big datasets should not be loading the entire `labels` and `pred_probs` arrays into memory, which is what is required for `find_label_issues_batched()`